target/linux/ramips/dts/ApexeraWiBot.dts

```diff
@@ -0,0 +1,170 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-wibot", "ralink,mt7620a-soc";
+	model = "Apexera WiBot";
+
+	chosen {
+		bootargs = "console=ttyS1,57600";
+        };
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		wan {
+			label = "wibot:blue:wan";
+			gpios = <&gpio1 5 1>;
+		};
+
+		wifi {
+			label = "wibot:blue:wifi";
+			gpios = <&gpio1 4 1>;
+		};
+
+		nand {
+			label = "wibot:blue:nand";
+			gpios = <&gpio1 3 1>;
+		};
+
+		boot {
```

Comment:
Add an `aliases {` node for the boot led. Check the other dts files for references.